conference-finder/src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import ConferenceItem from './ConferenceItem';
+import conferences from './conferenceDataMock.json';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ul>
+        <ConferenceItem conference={conferences[0]} />
```

Comment:
If you changed your mockData to a JavaScript file that exports an array you could simply map over that array and return a ConferenceItem each time:

<ul>{conferences.map(conference => <ConferenceItem conference={conference}>))}</ul>

Reply:
Similar to the way we mapped through the book array 2 weeks ago :)